src/components/MenuItemList.tsx

```diff
@@ -45,16 +59,23 @@ function MenuItemList({menuItems = [], shouldUseSingleExecution = false}: MenuIt
     return (
         <>
             {menuItems.map((menuItemProps) => (
-                <MenuItem
-                    key={menuItemProps.key ?? menuItemProps.title}
-                    onSecondaryInteraction={menuItemProps.link !== undefined ? (e) => secondaryInteraction(menuItemProps.link, e) : undefined}
-                    ref={popoverAnchor}
-                    shouldBlockSelection={!!menuItemProps.link}
-                    // eslint-disable-next-line react/jsx-props-no-spreading
-                    {...menuItemProps}
-                    disabled={!!menuItemProps.disabled || isExecuting}
-                    onPress={shouldUseSingleExecution ? singleExecution(menuItemProps.onPress) : menuItemProps.onPress}
-                />
+                <OfflineWithFeedback
+                    pendingAction={menuItemProps.pendingAction}
+                    onClose={menuItemProps.onPendingActionDismiss}
+                    errors={menuItemProps.error}
+                    shouldForceOpacity={menuItemProps.shouldForceOpacity}
+                >
```

Comment:
Is using `OfflineWithFeedback` here going to cause any weirdness in other menus?

Reply:
no because they won't be passing `children` will passed as they are